src/backend/web/renderers/team_renderer.py

```diff
@@ -188,10 +189,18 @@ def render_team_details(
                 AllianceHelper.get_alliance_details_and_pick_name(event, team.key_name)
             )
 
-            if alliance and "status" in alliance and "name" in alliance:
+            alliance_status = EventTeamStatusHelper._build_playoff_info(
+                team.key_name,
+                event.details,
+                event_matches,
```

Comment:
`_build_playoff_info` expects `event_matches` needs to be matches for the whole event, but i believe this only has matches for this current team?

Reply:
Since this is only generating the status based off of that team's alliance, I think that would be fine? I'll look more into the event team status helper.

Reply:
I think the edge case is if a team is a 3 pick and doesn't play any matches?

Reply:
I think the reason why we were generating this on the fly in the API rather than saving it in the database was so we could change the strings without going through crazy cash clearing

Reply:
I don't think it makes sense to store the strings in a database, but the various status dicts (espcially now that this dict is both used for team event statuses and the team detail pages) may make sense to store and/or cache in some way.